### PR TITLE
Feature/flho 346 2nd person name

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
Adding the (Shooting clubs) second person name page
https://jira.digital.homeoffice.gov.uk/browse/FLHO-346

### CODE REVIEW

File: apps/shooting-clubs/index.js [hunk 1]
```diff
@@ -14,7 +14,16 @@ module.exports = {
     '/activity': {
       fields: [
         'activity'
-      ]
+      ],
+      next: '/second-person-name'
+    },
+    '/second-person-name': {
+      fields: [
+          'second-person-name'
```
Comment (JoeChapman) [inline apps/shooting-clubs/index.js]: indentation

Comment (joefitter) [inline apps/shooting-clubs/index.js]: 🙀

**Comment by bendiggle** [comment]
@sulthan-ahmed Is this PR the same as this one https://github.com/UKHomeOffice/firearms/pull/76? Can this one be closed?

**Comment by sulthan-ahmed** [comment]
Yes @bendiggle

**Pull request closed without merging**